app/models/manageiq/providers/cloud_manager/vm.rb

```diff
@@ -308,6 +307,19 @@ def self.evacuate_queue(userid, vm, options = {})
     MiqTask.generic_action_with_callback(task_opts, queue_opts)
   end
 
+  def resize_queue(userid, options = {})
+    options[:src_ids] = [id]
+    # TODO: @requestId - figure out what to do. old code in resize:
+    #   if params[:id] && params[:id] != 'new'
+    #     @request_id = params[:id]
+    #   end
+    VmCloudReconfigureRequest.make_request(@request_id, options, current_user)
+  end
+
+  def current_user
+    @current_user ||= User.current_user
+  end
```

Comment:
I think it'd be better to pass in the `:user` as an option to `resize_queue` so it is more explicit, specifically caching this is not a good idea